cylc/flow/cfgspec/workflow.py

```diff
@@ -695,6 +690,11 @@ def get_script_common_text(this: str, example: Optional[str] = None):
                The runahead limit may be automatically raised if this is
                necessary to allow a future task to be triggered, preventing
                the workflow from stalling.
+
+            .. versionchanged:: 8.0.0
+
+               The deprecated ``[scheduling]max active cycle points`` setting
```

Comment:
But it is deprecated? https://github.com/cylc/cylc-flow/blob/9ed6035f2c8f02cf09a5672692b25ebf604cd7e3/cylc/flow/cfgspec/workflow.py#L1709-L1714

Reply:
The setting is deprecated, however, the functionality is not.

```
# deprecated
max active cycle points = 4

# fine
runahead limit = P4
```

Because we had previously marked `runahead limit` as deprecated this has caused some confusion.